### ISSUE TITLE
Narrator does not announce choice group is selected on navigating using Caps and arrow key

### ISSUE THREAD

**Issue by veloiya** [OfficeDev/office-ui-fabric-react#6880]
### Environment Information 
 
- __Browser and OS versions__: Edge + narrator
 
#### Actual behavior:
1. Go to https://developer.microsoft.com/en-us/fabric#/components/choicegroup.
2. Navigate to an unselected choice group with Caps + arrow keys.
3. Select the choice group with Caps + Enter.

Narrator does not announce that the choice group is selected.
 
#### Expected behavior:
 
Narrator should announce that the choice group has been selected after selection through Caps + Enter key.

#### Priorities and help requested:
   
Products/sites affected: Azure Devops

**Comment by JasonGore** [comment]
The behavior appears to be the following:
1. When using `CAPS + arrow` navigation, elements are announced both when radio button and text are highlighted.
2. When hitting `CAPS + enter`, you here a selected tone, but no announcement of selected or unselected.

However, I've also found [several](https://www.w3.org/TR/wai-aria-practices/examples/radio/radio-2/radio-2.html) [examples](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html) of ARIA radiogroup examples that exhibit the same behavior. If this is expected behavior, it seems as if Narrator is at fault.

I'd recommend following up with [Narrator team](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Accessibility-Troubleshooting) regarding this issue. Thanks!
